spec/models/user_spec.rb

```diff
@@ -6,22 +6,57 @@
   it {should validate_presence_of(:full_name)}
   it {should validate_uniqueness_of(:email)}
 
-  it {should have_many(:reviews)}
+  it {should have_many(:reviews).order('created_at DESC')}
   it {should have_many(:queue_items).order('position asc')}
```

Comment:
I highly recommend using spaces around operators and possibly symbols instead of strings in `.order` syntax - `it { should have_many(:reviews).order(created_at: :desc) }`